guides/dev/package_managers/README.md

```diff
@@ -28,6 +28,41 @@ It is thus recommended to use of the following strategies:
 
 Both options are explained in more detail in the [Setting CKEditor Configuration](http://docs.ckeditor.com/#!/guide/dev_configuration) article.
 
+## npm
+
+This article assumes that you have **npm** already installed and added to your `PATH`. If this is not the case, please follow the [installation guide](https://docs.npmjs.com/getting-started/installing-node).
+
+### Usage
+
+In order to fetch the most recent build execute the following command:
+
+	npm install ckeditor
+
+By default CKEditor will be placed in the `node_modules/ckeditor` directory.
+
+### Adding CKEditor as Dependency Using the `package.json` File
```

Comment:
You are right, using package.json file to manually add dependencies it's not a common case or default usage so this part is not crucial and removing it will make docs much simpler and cleaner. Not sure about bower but it is probably a similar case.